src/cases/containers/index.ts

```diff
@@ -24,6 +25,7 @@ export const containers: any[] = [
   CaseShareComponent,
   CaseShareConfirmComponent,
   CaseShareCompleteComponent,
+  RoleAccessComponent,
```

Comment:
Finicky - could have sorted these while you were in this area

Reply:
This is not going to be checked in. This is for Test only purpose. Once @rayliangatos  has the new module. All these components will be moved to that module. Thanks.

Reply:
They could have been moved ordered. Anyhow - finicky comment.